koku/masu/processor/__init__.py

```diff
@@ -143,3 +143,10 @@ def is_ingress_rate_limiting_disabled():  # pragma: no cover
     if res:
         LOG.info(log_json(msg="ingress rate limiting disabled"))
     return res
+
+
+def override_customer_group_by_limit(account):  # pragma: no cover
+    """Flag the customer to override max group_by count."""
+    account = convert_account(account)
+    context = {"schema": account}
+    return UNLEASH_CLIENT.is_enabled("cost-management.backend.override_customer_group_by_limit", context)
```

Comment:
Given that this is an API feature flag, I think from an organizational standpoint, this func should live...not in Masu. But, idk a better place to put it.

Reply:
We could just move all these helpers into:
https://github.com/project-koku/koku/blob/main/koku/koku/feature_flags.py